app/src/main/java/org/wikipedia/main/MainActivity.kt

```diff
@@ -160,6 +165,14 @@ class MainActivity : SingleFragmentActivity<MainFragment>(), MainFragment.Callba
         binding.mainToolbar.elevation = 0f
     }
 
+    @VisibleForTesting
+    fun getIdlingResource(): IdlingResource? {
+        if (mIdlingResource == null) {
+            mIdlingResource = CountingIdlingResource("GLOBAL")
+        }
+        return mIdlingResource
```

Comment:
Czy mogę dopisywać metody zwracające IdlingResource do kodu źródłowego? Czy raczej próbować to ograć w jakiś inny sposób? (Chyba żeby dać sobie z IdlingResouce spokój i waity rozwiązać inaczej)

Reply:
No i tutaj dochodzimy to tematu o którym chciałbym pogadać na dzisiejszym spotkaniu - generalnie IdlingResource to zło i unikaj jak się da. Pogadamy na callu dokładniej dlaczego